src/MonteCarloMoves/nve_walks.jl

```diff
@@ -178,25 +24,24 @@ Returns:
 - `e_demon::Float64`: The updated energy of the demon.
 """
 function single_atom_demon_walk!(
```

Comment:
@yangmr04, I'm OK with including the demon walks in the `dev` branch, but let's not release it until we can convince ourselves that it does something more accurately or efficiently than the other sampling algorithms.

Reply:
Agreed! I will move them out to an "experimental" branch later.